janis_core/tests/test_translation_cwl.py

```diff
@@ -16,7 +16,7 @@
     OperatorResourcesTestTool,
 )
 
-import cwl_utils.parser_v1_0 as cwlgen
+import cwl_utils.parser_v1_2 as cwlgen
```

Comment:
Maybe we can put this import line `import cwl_utils.parser_v1_2 as cwlgen` in one of our modules and all other code in the janis_core can just `import cwlgen` from that module. 
So, next time, you will only need to update from one place?